### PR TITLE
Team 4 Files added

### PR DESCRIPTION
**Pull request by KashyapPavra**

### CODE REVIEW

File: teams/4/page-2.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+              <div class="red"><a href="default.asp"><img src="smiley.gif" alt="img" ></a>
+                <a href="default.asp"><img src="smiley.gif" alt="img" ></a>
+                    <a href="default.asp"><img src="smiley.gif" alt="img" ></a>
+                        <a href="default.asp"><img src="smiley.gif" alt="img" ></a>
+                            <a href="default.asp"><img src="smiley.gif" alt="img" ></a>
+                                <a href="default.asp"><img src="smiley.gif" alt="img" ></a>
+                                    <a href="default.asp"><img src="smiley.gif" alt="img" ></a>
+                                        <a href="default.asp"><img src="smiley.gif" alt="img" ></a>
+
+            </div></div>
```
Comment (nikabutikashvili) [inline teams/4/page-2.html:22]: What for is this div?

File: teams/4/page-2.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+</head>
+<body>
+    <h1 class="heady"></h1>
+    <div class="sidebar">
+        <div class="float-parent-element">
+            <div class="float-child-element">
+
+              <div class="red"><a href="default.asp"><img src="smiley.gif" alt="img" ></a>
+                <a href="default.asp"><img src="smiley.gif" alt="img" ></a>
+                    <a href="default.asp"><img src="smiley.gif" alt="img" ></a>
```
Comment (nikabutikashvili) [inline teams/4/page-2.html:15]: I think the file needs prettier

File: teams/4/page-2.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+<body>
+    <h1 class="heady"></h1>
```
Comment (nikabutikashvili) [inline teams/4/page-2.html:8]: For which text is this heading? As far as I see, in my opinion, "Billing" should be H1 in this page, but in that case this tag should no be here structurally.

File: teams/4/page-2.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
```
Comment (nikabutikashvili) [inline teams/4/page-2.html:1]: General comments about this page. Use sections instead of divs everywhere.

**Review by nikabutikashvili — APPROVED** [review]

**Review by wasim919 — APPROVED** [review]

**Review by wriozumi — APPROVED** [review]

**Review by Giancarix117 — APPROVED** [review]

**Pull request merged**